TestData/test2-happy-path.json

```diff
@@ -0,0 +1,84 @@
+{
+  "Participants": [
+    {
+      "nhs_number": "1111111112",
+      "next_test_due_date": "null",
+      "gp_practice_id": "39",
+      "subject_status_code": "NORMAL",
+      "is_higher_risk": "null",
+      "higher_risk_next_test_due_date": "null",
+      "removal_reason": "null",
+      "removal_date": "null",
+      "bso_organisation_id": "null",
+      "early_recall_date": "null",
+      "latest_invitation_date": "null",
+      "preferred_language": "null",
+      "higher_risk_referral_reason_code": "null",
+      "date_irradiated": "null",
+      "is_higher_risk_active": "null",
+      "gene_code": "null",
+      "ntdd_calculation_method": "null"
+    },
+    {
+      "nhs_number": "1111111110",
+      "next_test_due_date": "null",
+      "gp_practice_id": "null",
+      "subject_status_code": "NORMAL",
+      "is_higher_risk": "null",
+      "higher_risk_next_test_due_date": "null",
+      "removal_reason": "MENTAL_HOSPITAL",
+      "removal_date": "2017-07-28",
+      "bso_organisation_id": "null",
+      "early_recall_date": "null",
+      "latest_invitation_date": "null",
+      "preferred_language": "null",
+      "higher_risk_referral_reason_code": "null",
+      "date_irradiated": "null",
+      "is_higher_risk_active": "null",
+      "gene_code": "null",
+      "ntdd_calculation_method": "null"
+    }
+  ],
+  "Episodes": [
+    {
+    "episode_id": "123456",
+    "participant_id": "ABC123",
+    "screening_id": "XYZ789",
+    "nhs_number": "1234567890",
+    "episode_type_id": "ET001",
+    "episode_open_date": "2022-01-01",
+    "appointment_made_flag": "YES",
+    "first_offered_appointment_date": "2022-01-15",
+    "actual_screening_date": "2022-02-01",
+    "early_recall_date": "2022-03-01",
+    "call_recall_status_authorised_by": "John Doe",
+    "end_code_id": "EC001",
+    "end_code_last_updated": "2022-04-01",
+    "appointment_made": "YES",
+    "organisation_id": "ORG123",
+    "batch_id": "BAT456",
+    "record_insert_datetime": "2022-05-01 12:00:00",
+    "record_update_datetime": "2022-06-01 12:00:00"
```

Comment:
The properties names should match what is shown [here](https://nhsd-confluence.digital.nhs.uk/display/DTS/Release+0+-+Service+Insights+-+BI+and+Data+Analytics+product+-+MVP+-+Data+Design#Release0ServiceInsightsBIandDataAnalyticsproductMVPDataDesign-ServiceOrientedDataDesign).

Not all of the properties that will be mapped to Episode entity model will come from BS Select, properties like participant_id should not be included here.

Reply:
Updated properties to include only those from BS Select